src/Slices/ServiceDetails/UI/Wrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { PageContainer } from "@/UI/Components";
+import { words } from "@/UI/words";
+
+interface Props {
+  name: string;
```

Comment:
you will need to add the children here 
`children?: React.ReactNode (array if multiple are an option)`